coffee_core/src/config.rs

```diff
@@ -46,8 +46,10 @@ impl CoffeeConf {
             def_path = data_dir.to_owned();
         }
 
-        // FIXME: check for double slash
-        def_path += "/.coffee";
+        if !def_path.ends_with('/') {
+            def_path += "/";
+        }
```

Comment:
```suggestion
        def_path = def_path.strip_suff('/').unwrap_or(def_path);
```

This is a better way to write this in rust? or maybe it is just my preference!